### PR TITLE
Or 1086 titond backend add relayer to component api unittest

### PR DESCRIPTION
**Pull request by nguyenzung**
- Add relayer
- Add unittest

Thank you!

### CODE REVIEW

File: pkg/api/relayer.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	}
+	fmt.Printf("Created Relayer Deployment: %s\n", deployment.GetName())
+
+	err = t.k8s.WaitingDeploymentCreated(deployment.Namespace, deployment.Name)
+	if err != nil {
+		return err
+	}
+	relayer.Status = true
+
+	obj, _ = kubernetes.BuildObjectFromYamlFile("./deployments/relayer/ingress.yaml")
```
Comment (rlgns98kr) [inline pkg/api/relayer.go:87]: Oh, we don't have to relayer ingress!
I'm sorry for my lack of communication. Thank you!

File: deployments/relayer/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
```
Comment (rlgns98kr) [inline deployments/relayer/ingress.yaml:1]: Okay, we can remove this yaml file(Ingress) for relayer! 🙇🏼‍♂️

Comment (nguyenzung) [reply deployments/relayer/ingress.yaml]: I removed the ingress file

File: pkg/api/relayer.go (hunk 1 above)
Comment (nguyenzung) [reply pkg/api/relayer.go]: Thank you!
I updated

**Review by rlgns98kr — APPROVED** [review]
Thank you very much, Brave! 😍

**Pull request merged**